winbuild/Makefile.vc

```diff
@@ -21,10 +22,20 @@ CFGSET=true
 !MESSAGE                                  Libraries can be fetched at http://pecl2.php.net/downloads/php-windows-builds/
 !MESSAGE                                  Uncompress them into the deps folder.
 !MESSAGE   WITH_SSL=<dll or static>     - Enable OpenSSL support, DLL or static
+!MESSAGE      SSL_LIBS=<File1 File2>    - OpenSSL libs to use (default: libeay32.lib ssleay32.lib) (requires WITH_SSL)
+!MESSAGE      SSL_HEADER_PATH=<path>    - Path to OpenSSL include files. (requires WITH_SSL)
```

Comment:
Yeah, names are just names, new scheme could be introduced. I'd just see no need, as keeping current were already associative. Base on these
- DEVEL_INCLUDE exists, consequently XXX_INCLUDE
- DEVEL_LIB exists, consequently XXX_LIB
- XXX_LIBS exists, fe SSL_LIBS, yep, can be reused to be put from outside.

OFC, can explicitly append "_PATH", but seems excessive. I think we better discuss this once, also to ensure the names are concise and scheme is reused where possible.